src/clip.sol

```diff
@@ -210,11 +210,11 @@ contract Clipper {
     // Could get this from rmul(Vat.ilks(ilk).spot, Spotter.mat()) instead, but
     // if mat has changed since the last poke, the resulting value will be
     // incorrect.
-    function getPrice() internal returns (uint256 price) {
+    function feed() internal returns (uint256 feed) {
```

Comment:
`feed` seems like it would denote an oracle itself, not a price. If we're doing meme naming, I'd invent a new meme here, ...maybe `mark`, derived from "market price". cc @gbalabasquer

Reply:
I gave my approval to the other PR. At this point I'd prefer we focus in the other PRs than thinking on this variable naming. The other PR looks good to me.